<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>578</width>
    <height>368</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="user">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>5</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>사용자 :</string>
   </property>
  </widget>
  <widget class="Line" name="line_22">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>531</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="user_widget">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>6</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="add_costmok_button">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>120</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>등록</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>541</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="costmok_cancel_button">
   <property name="geometry">
    <rect>
     <x>409</x>
     <y>310</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>닫기</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="costmok_save_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>220</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>저장(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>83</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>지출 항</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="new_costmok_widget">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>80</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableWidget" name="registed_costmok_tableWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>181</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>기존 등록된 목</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>id</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="new_costmok_tableWidget">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>110</y>
     <width>151</width>
     <height>241</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <column>
    <property name="text">
     <string>신규등록 목</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>15</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 지출 목  등록</string>
   </property>
  </widget>
  <widget class="QComboBox" name="costhang_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>80</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>292</x>
     <y>84</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>신규등록 지출 목</string>
   </property>
  </widget>
  <widget class="QComboBox" name="gubun_comboBox_widget">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>54</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>57</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>회계 구분</string>
   </property>
  </widget>
  <widget class="QPushButton" name="costmok_delete_button">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>170</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 항 수정</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="costmok_delete_button_2">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>선택 항 삭제</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>gubun_comboBox_widget</tabstop>
  <tabstop>costhang_comboBox_widget</tabstop>
  <tabstop>new_costmok_widget</tabstop>
  <tabstop>add_costmok_button</tabstop>
  <tabstop>costmok_save_button</tabstop>
  <tabstop>costmok_cancel_button</tabstop>
  <tabstop>registed_costmok_tableWidget</tabstop>
  <tabstop>new_costmok_tableWidget</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>costmok_save_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costmok_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>466</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>556</x>
     <y>269</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costmok_cancel_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costmok_save_cancel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>321</y>
    </hint>
    <hint type="destinationlabel">
     <x>576</x>
     <y>322</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>add_costmok_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costmok_reg_input()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>489</x>
     <y>126</y>
    </hint>
    <hint type="destinationlabel">
     <x>407</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costmok_delete_button</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>costmok_modify()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>500</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>535</x>
     <y>176</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>costmok_delete_button_2</sender>
   <signal>clicked()</signal>
   <receiver>Dialog</receiver>
   <slot>delete_selected_row()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>444</x>
     <y>276</y>
    </hint>
    <hint type="destinationlabel">
     <x>400</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>costmok_reg_input()</slot>
  <slot>costmok_save()</slot>
  <slot>costmok_save_cancel()</slot>
  <slot>registed_mok()</slot>
  <slot>delete_selected_row()</slot>
  <slot>costmok_modify()</slot>
 </slots>
</ui>
